composer.json

```diff
@@ -12,7 +12,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3.0",
+        "psr/container": "^1.0",
+        "phpunit/phpunit": "5.0.0"
```

Comment:
See https://github.com/silexphp/Pimple/pull/216#discussion_r101743206

@Sam-Burns already said he'll take it out. I agree, because this is not relevant to this PR.